ddtrace/settings.py

```diff
@@ -57,3 +58,58 @@ def _add(self, integration, settings):
         """
 
         self._config[integration] = deepcopy(settings)
+
+
+class HttpConfig(object):
+    """Configuration object that expose an API to set and retrieve both global and integration specific settings
+    related to the http context.
+    """
+
+    _traced_headers = 'traced_headers'
+
+    def __init__(self):
+        self._integrations_config = {}
+        self._global_config = {}
```

Comment:
I think an alternative idea could be to use a single config object, and use `None` as the integration key for "global".

e.g.

```python
self._config = {
    None: {
        'traced_headers': set([]),
    },
    'falcon': {
        'traced_headers': set([]),
    },
}
```

It'll make `_set_config_key` a little easier as well:

```python
def _set_config_key(self, integration, key, value):
    if integration not in self._config:
        self._config[integration] = {}
    self._config[integration][key] = value
```

Reply:
This is obsolete as we went for the `IntegrationConfig` approach